ext/coms.c

```diff
@@ -158,6 +162,13 @@ bool ddtrace_coms_minit(size_t initial_stack_size, size_t max_stack_size, size_t
     atomic_store(&ddtrace_coms_globals.next_group_id, 1);
     atomic_store(&ddtrace_coms_globals.current_stack, stack);
 
+    ddog_Option_VecU8 writer_result = ddog_create_agent_remote_config_writer(&dd_agent_config_writer, &ddtrace_coms_agent_config_handle);
```

Comment:
Not the fault of this PR, but noticed that `MaybeError` is a leaky abstraction when translated to C, as this should be `ddog_MaybeError` and the tag should be something like `DDOG_MAYBEERROR_SOME` or something. Makes it a bit confusing, because when initially reading the code, I was thinking... why do they call `dog_MaybeError_drop` when they have `ddog_Option_VecU8` instead?

Anyway, not something that needs solved in this PR.

Reply:
I did not author that abstraction and am not too happy with it, but that's what we currently have :-D
I'm open to addressing it in a future change.